common/changes/alert-dialog_2017-02-20-10-12.json

```diff
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Dialog: Accessibility fix for confirmation dialogs",
+      "type": "patch"
+    },
+    {
+      "comment": "",
+      "packageName": "@uifabric/utilities",
+      "type": "none"
+    },
+    {
+      "comment": "",
+      "packageName": "@uifabric/example-app-base",
+      "type": "none"
+    }
+  ],
+  "email": "manishda@microsoft.com"
```

Comment:
Why are so many changes in this PR? Was expecting it to just the switch to CSS modules.